rust/xaynet-analytics/src/data_combination/data_points/was_active_past_n_days.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{
+    data_combination::data_points::data_point::{
+        CalcWasActivePastNDays,
+        CalculateDataPoints,
+        DataPointMetadata,
+    },
+    data_provision::analytics_event::AnalyticsEvent,
+};
+
+impl CalcWasActivePastNDays {
+    pub fn new(metadata: DataPointMetadata, events: Vec<AnalyticsEvent>) -> CalcWasActivePastNDays {
+        CalcWasActivePastNDays { metadata, events }
+    }
+}
+
+impl CalculateDataPoints for CalcWasActivePastNDays {
+    fn metadata(&self) -> DataPointMetadata {
+        self.metadata
+    }
+
+    fn calculate(&self) -> Vec<u32> {
+        if self.events.is_empty() {
```

Comment:
Give the previous comment this can be also written 
`vec![!self.events.is_empty() as u32]`

Reply:
will do, thanks